### DISCUSSION TITLE
[documentation] Raspberry Pi and NixOS: Setup FreshRSS

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#6394] [Q&A]
maybe it will help someone, who wants to setup Raspberry Pi and NixOS.

Found via https://mastodon.social/@msfjarvis@androiddev.social/112338318414555142

Http auth: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/freshrss-http-auth.nix
none auth: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/freshrss-none-auth.nix
PostgreSQL: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/freshrss-pgsql.nix
Sqlite: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/freshrss-sqlite.nix

**Comment by Alkarex** [comment]
Maybe consider moving to https://github.com/FreshRSS/FreshRSS/discussions/categories/q-a

**Comment by math-GH** [comment]
would it be worth to add it to our documentation (help center)?

**Comment by Alkarex** [comment]
Possibly, if there is enough substance for a page about it, but I am also worried about maintenance.
I see https://github.com/FreshRSS/FreshRSS/discussions/categories/q-a as a more live knowledge base for a diversity of things
